api/src/main/resources/org/openmrs/api/db/hibernate/Diagnosis.hbm.xml

```diff
@@ -21,13 +21,16 @@
 			<generator class="native" />
 		</id>
 
-		<property name="diagnosis" column="coded_or_free_text" length="50" not-null="true">
-			<type name="org.openmrs.CodedOrFreeText"/>
-		</property>
+		<component name="diagnosis" class="org.openmrs.CodedOrFreeText">
+			<many-to-one name="coded" class="org.openmrs.Concept" column="diagnosis_coded" />
+			<property name="nonCoded" type="java.lang.String" length="255" column="diagnosis_non_coded" />
+			<many-to-one name="specificName" class="org.openmrs.ConceptName" column="diagnosis_coded_name" />
+		</component>
 
 		<property name="certainty" column="condition_verification_status" length="50" not-null="true">
```

Comment:
Do we have a column named condition_verification_status in the liquibase table creation changeset?